src/style.css

```diff
@@ -1,3 +1,64 @@
 body {
   margin: 0;
 }
+
+.header {
+  display: flex;
+  width: 100%;
+  padding: 0 50px;
+  box-sizing: border-box;
+}
+
+.logo {
+  margin: 10px 0;
+  height: 40px;
+}
+
+.list {
+  list-style-type: none;
+  display: flex;
+  margin: 0;
+  padding: 0;
+  height: 60;
+  justify-content: center;
+}
+
+.navItem {
+  text-decoration: none;
+  color: #000;
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+
+  font-weight: 500;
+  line-height: 60px;
+  text-transform: uppercase;
+}
+
+.listItem {
+  margin: 0 10px;
```

Comment:
It's better to use margin from Figma, so you can set margin-right: 20px and remove this margin from the last link using :last-child